app/modules/site_settings/models.py

```diff
@@ -116,38 +117,33 @@ class SiteSetting(db.Model, Timestamp):
         'transloaditKey': {
             'type': str,
             'public': False,
-            'default': '',
-            'isApiKey': True,
+            'default': lambda: current_app.config.get('TRANSLOADIT_KEY'),
         },
         'transloaditTemplateId': {
             'type': str,
             'public': False,
-            'default': '',
-            'isApiKey': True,
+            'default': lambda: current_app.config.get('TRANSLOADIT_TEMPLATE_ID'),
         },
         'transloaditService': {
             'type': str,
             'public': False,
-            'default': '',
-            'isApiKey': True,
+            'default': lambda: current_app.config.get('TRANSLOADIT_SERVICE'),
         },
         'googleMapsApiKey': {
             'type': str,
-            'public': False,
-            'default': '',
-            'isApiKey': True,
+            'public': True,
+            'default': lambda: current_app.config.get('GOOGLE_MAPS_API_KEY'),
         },
         'sentryDsn': {
             'type': str,
-            'public': False,
-            'default': '',
-            'isApiKey': True,
+            'public': True,
+            'default': lambda: current_app.config.get('SENTRY_DSN'),
         },
         'flatfileKey': {
             'type': str,
-            'public': False,
-            'default': '',
-            'isApiKey': True,
+            'public': True,
+            'permission': lambda: not current_user.is_anonymous,
```

Comment:
Only a researcher should be able to do a bulk upload (and I think this is only neccessary for bulk upload) so this should be is_researcher